text/0000-deprecate-string-based-actions.md

```diff
@@ -0,0 +1,89 @@
+- Start Date: 2020-05-23
+- Relevant Team(s): Ember.js
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# Deprecate String Based Actions
+
+## Motivation
+
+> In Ember 1.13 we introduced [closure actions](https://github.com/emberjs/rfcs/blob/00ac2685c86f27d41547012903f485a4ef338d27/active/0000-improved-actions.md) have been recommended over string based actions. This officially deprecates string based actions in favor of closure actions,
+which provide many benefits such as
+
+1. simpler passing through components
+2. less need for an actions hash
+3. ability to return a value
+4. typing in TypeScript
+5. compatible with Glimmer Components
+
+## Detailed Design
+
+We will deprecate the following:
+
+1. The `send` api.
+2. The use of a string parameter to the `action` helper.
+
+## Transition Path
+
+Users will transition to closure actions.
+
+From:
+
+```js
+export default Component.extend({
+    someMethod() {
+        this.send('actionName');
+    }
+
+    actions: {
+        actionName() {
+            // do something
+        }
+    }
+});
+```
+
+To:
+
+```js
+export default Component.extend({
+    someMethod() {
+        this.actionName();
+    }
```

Comment:
this is invalid syntax, when using `.extend` you'd need a comma here

Reply:
correct. Thanks.